lib/libc/newlib/include/stdint.h

```diff
@@ -13,12 +13,15 @@
  */
 
 #include <newlib.h>
+#include <stddef.h>
 
 #ifdef __NEWLIB__
 /* Has this header.  Older versions do it in <stdint.h>. */
 #include <sys/_stdint.h>
 #endif /* __NEWLIB__ */
 
+void qsort_r(void *base, size_t nmemb, size_t size,
```

Comment:
This should probably be in stdlib.h instead of stdint.h; that's the header referenced in the qsort_r man page.

Reply:
oops, saw std... and didnt even see the int ;).  Will fix.

Reply:
fixed